site/content/user-guide/java_client_sender.md

```diff
@@ -3,308 +3,87 @@ title = "Java - integrate Hono sender"
 weight = 220
 +++
 
-Simple example of a Java main program that sends telemetry data to the default tenant from the registered default device. 
+To illustrate how Hono can be integrated with Java code, a simple program is provided that sends telemetry or event data 
+to the default tenant from a device. 
 It shall serve as a blueprint to integrate your existing java source code with Hono. 
-<!--more-->
 
+The code is found in the example module in the package `org.eclipse.hono.vertx.example`.
```

Comment:
we should format the module name:
The code can be found in the `example` module in package `org.eclipse...`